views/components/GraphTable/helpers.js

```diff
@@ -20,3 +23,13 @@ const sortAllSitesBeforeTotalsSite = (siteA, siteB) => {
   if (siteA.name === TOTALS) return -1
   if (siteB.name === TOTALS) return -1
 }
+
+export const formatSiteData = (studyCounts, studyTargets, studyPercentage) =>
```

Comment:
🎱 I like using objects for arguments. It lets the caller pass the args as named parameters and order doesn't matter 
```suggestion
export const formatSiteData = ({ studyCounts, studyTargets, studyPercentage }) =>
```

usage
```diff
- const formattedSiteData = formatSiteData(12, 36, 33.33)
+ const formattedSiteData = formatSiteData({
+   studyCounts: 12,
+   studyTargets: 36,
+   studyPercentage: 33.33
+ })
```